srcpkgs/session-desktop/files/session.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Session
+GenericName=Messenger
+Comment=Session is an end-to-end encrypted messenger.
+Icon=/usr/share/icons/hicolor/1024x1024/apps/session.png
```

Comment:
Icon=session should be enough, the DE will then select a prefered size and format.

Reply:
Yes that what I thought. But as it have only one size, on the DE (at least Plasma for me) there's just no icon shown 🤷‍♀️

Reply:
`/usr/share/icons/hicolor/scalable/apps/`

Isn't upstream providing a desktop file? Don't they have a Makefile which installs everything?

Reply:
They don't, it's built with yarn (the build system is basically the same as signal-desktop as it's a *now pretty much different* fork). I also modified to generate some png.